docs/modules/ROOT/pages/references/storage/gcp.adoc

```diff
@@ -0,0 +1,60 @@
+= Storage options on Google Cloud Platform
+
+== Read write once block disk storage
+
+OpenShift 4 brings along a storage provisioner which taps into the disk storage offering of the Google Compute Engine.
+The following storage classes are provided:
+
+[horizontal]
+standard:: Zonal standard persistent disks—Efficient and reliable block storage
+regional:: Regional standard persistent disks—Efficient and reliable block storage with synchronous replication across two zones in a region
+ssd:: Zonal SSD persistent disks—Fast and reliable block storage
+regional-ssd:: Regional SSD persistent disks—Fast and reliable block storage with synchronous replication across two zones in a region
+
+Check https://cloud.google.com/compute/docs/disks/[Google Compute Engine Storage Options] for the details on that storage.
+
+Check https://cloud.google.com/compute/disks-image-pricing#disk[Google Compute Engine Disk Pricing] for the pricing of the storage options.
+By default, we host clusters in Zürich (europe-west6).
```

Comment:
I would not mention this here as it's already being mentioned in the product documentation and I'm sure that this will / could diverge in the future.